miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/MenuController.java

```diff
@@ -248,6 +248,12 @@ public ModelAndView myAccountMenu(ModelMap model) {
     }
   }
 
+  @GetMapping("/")
+  public ModelAndView redirectMisoRoot(ModelMap model) {
+    model.clear();
+    return new ModelAndView("redirect:mainMenu", model);
+  }
```

Comment:
Can this also be applied to endpoint `""`? `http://localhost/miso` without the trailing slash still throws the 404.

Reply:
I tried, but it doesn't work. I don't think there's a way to map `/miso` because of the weird way that that portion of the url is forced in. I do think that a redirect from `/miso/` makes more sense than redirecting from `/miso` anyway though

Reply:
Yeah, that weird URL forcing is regrettable. I do think that people are more likely to try to go to `/miso` than `/miso/`, but I don't think there's anything to be done about that. :disappointed: